ultra/MANIFEST.in

```
@@ -1 +1,5 @@
 include sac/checkpoint/checkpoint
+include ultra/agent_pool.json
+include config.yaml
+include ultra/scenarios/task1/config.yaml
+include ultra/scenarios/task2/config.yam
```

Oh wait, perhaps should be config.yaml?
```suggestion
include ultra/scenarios/task2/config.yaml
```